include/ps/internal/message.h

```diff
@@ -129,6 +129,7 @@ struct Node {
   /** \brief the length of the endpoint name */
   size_t endpoint_name_len = 0;
   /** \brief the auxilary id. currently used for fabric van communication setup */
+  /** and used for initilization phrase for p2p communication**/
   int aux_id;
```

Comment:
is this an id or rank? looks like it's used as a rank.

Reply:
Yes it's used as a rank. But i think reusing the field is fine since it's "auxiliary" and is different from the real id.